soundrts/lib/collision.py

```diff
@@ -30,19 +30,19 @@ def __init__(self, xmax, res):
         self._set = set()
         self.xmax = xmax
         self.res = res
-        self.amax = self.xmax / self.res
+        self.amax = self.xmax // self.res
```

Comment:
I am sure that this should work for some time, ut I've found some strange numbers like a float of 9.2 and a result int of 5000000. I got stuck with these stranger situations.